source/uri-options/tests/tls-options.yml

```diff
@@ -0,0 +1,64 @@
+tests:
+    -
+        description: "Valid required tls options are parsed correctly"
```

Comment:
I assume testing that SRV schemes default tls to true may be outside the scope of this spec? I imagine that might also be a bit complicated, as you'd have to ensure the SRV record itself was valid just to run such a test.

Reply:
I'm not quite sure what that test would even look like in the current format. I think that marking that `tls` is expected to be true based on `mongodb+srv` assumes that drivers implement the decision about whether to enable TLS implicitly in the URI parsing, which I don't think is something we want to mandate in the spec.

Reply:
Fair enough. Drivers could very well enforce that default when resolving SRV records and incorporate it into the URI options returned from the records.

Also, per discussion in #drivers Slack last week, I think a few drivers actually try to resolve SRV records earlier than when we'd logically expect MongoClient to perform _any_ IO (and initialize SDAM), so best to leave this untested here for maximum compatibility. Otherwise, we might require a valid SRV record exist just to run what should be unit tests.

Reply:
Given the discussion, I'm going to keep things as they are now in the tests with regards to SRV (i.e. not testing it in this spec).